parquet-hadoop/src/main/java/org/apache/parquet/crypto/AesGcmDecryptor.java

```diff
@@ -51,7 +51,11 @@ public byte[] decrypt(byte[] lengthAndCiphertext, byte[] AAD) {
   public byte[] decrypt(byte[] ciphertext, int cipherTextOffset, int cipherTextLength, byte[] AAD) {
 
     int plainTextLength = cipherTextLength - GCM_TAG_LENGTH - NONCE_LENGTH;
-    if (plainTextLength < 1) {
```

Comment:
AES GCM decryption is able to handle an empty plaintext. It will also use the key to check the IV/tag integrity.
So the fix can be just replacement of  `if (plainTextLength < 1)` { with `if (plainTextLength < 0) {`

Reply:
Thanks for the suggestions. Updated.